### PR TITLE
Add new setting for payment confirmation email

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@caesar2164 @kluo Please review.

**Comment by stvstnfrd** [comment]
retest this please

File: lms/envs/aws.py [hunk 1]
```diff
@@ -191,6 +191,13 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_CONFIRM_EMAIL = ENV_TOKENS.get(
+    'PAYMENT_CONFIRM_EMAIL',
+    ENV_TOKENS.get(
+        'PAYMENT_SUPPORT_EMAIL',
+        PAYMENT_CONFIRM_EMAIL
```
Comment (caesar2164) [inline lms/envs/aws.py]: Why the crazy double level .get here? Are you trying to coalesce the two values?

**Comment by caesar2164** [comment]
@stvstnfrd - this will eventually be 2 commits, right? 1 for `PAYMENT_CONFIRM_EMAIL` and one for the PEP8 fix?

File: lms/envs/aws.py (hunk 1 above)
Comment (stvstnfrd) [inline lms/envs/aws.py]: Yeah, that's the gist. It's a bit more clear starting from `lms/envs/common.py`, where you can see that we want `PAYMENT_CONFIRM_EMAIL` to default to `PAYMENT_SUPPORT_EMAIL`. Then to accomplish the same thing here, we'll
- try to use the value of `*_CONFIRM_*` from JSON
- try to use the value of `*_SUPPORT_*` from JSON
- fall back to the value of `*_CONFIRM_*`
- default to value of `*_SUPPORT_*` (implicitly from `lms/envs/common.py`)

Does that make sense?

**Comment by kluo** [comment]
+1

**Comment by stvstnfrd** [comment]
@caesar2164 This will ultimately be a single commit; the PEP8 changes have been rebased out.

File: lms/envs/aws.py (hunk 1 above)
Comment (caesar2164) [inline lms/envs/aws.py]: yep, that's what I thought

**Comment by caesar2164** [comment]
coolio, I'm :+1:

**Pull request merged**